### PR TITLE
ADCM-2903_1 add ui test for ldap

### PR DESCRIPTION
**Pull request by annademina**

### CODE REVIEW

File: tests/ui_tests/app/page/common/tooltip_links/page.py [hunk 1]
```diff
@@ -51,6 +51,30 @@ def run_action(self, tab_name: str, action_name: str):
         self.find_and_click(ActionDialog.run)
         self.wait_element_hide(ActionDialog.body)
 
+    @allure.step("Check action {action_name} in ADCM tab is inactive")
+    def is_adcm_actions_inactive(self, action_name: str):
```
Comment (Sealwing) [inline tests/ui_tests/app/page/common/tooltip_links/page.py:55]: As you check one aciton, I suggest naming it `is_adcm_action_inactive`

Is ADCM action button unique for ADCM?
I mean we have the similar button on detailed Cluster page, for example

File: tests/ui_tests/app/page/common/tooltip_links/page.py [hunk 2]
```diff
@@ -51,6 +51,30 @@ def run_action(self, tab_name: str, action_name: str):
...
+    def run_adcm_action(self, action_name: str):
+        """Run Action from toolbar"""
+
+        self.wait_element_visible(CommonToolbarLocators.admin_link)
+        self.find_and_click(CommonToolbarLocators.adcm_action_btn)
+        self.wait_element_visible(CommonToolbarLocators.Popup.popup_block)
+        self.wait_element_clickable(CommonToolbarLocators.Popup.item(action_name)).click()
+        self.wait_element_visible(ActionDialog.body)
+        self.find_and_click(ActionDialog.run)
+        self.wait_element_hide(ActionDialog.body)
```
Comment (Sealwing) [inline tests/ui_tests/app/page/common/tooltip_links/page.py:76]: `run_action` isn't suitable here?

**Review by Sealwing — CHANGES_REQUESTED** [review]

File: tests/ui_tests/app/page/common/tooltip_links/page.py (hunk 1 above)
Comment (annademina) [reply tests/ui_tests/app/page/common/tooltip_links/page.py:55]: no, they are different

Comment (annademina) [reply tests/ui_tests/app/page/common/tooltip_links/page.py]: renamed

**Review by Sealwing — APPROVED** [review]

**Pull request merged**